app/uk/gov/hmrc/tradergoodsprofilesrouter/models/audit/request/AuditUpdateRecordRequest.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tradergoodsprofilesrouter.models.audit.request
+
+import play.api.libs.json.{Json, OFormat}
+
+import java.time.Instant
+
+case class AuditUpdateRecordRequest(
```

Comment:
this is the same as the AuditCreateRecordRequest. You could reuse the same one for both create and Update audit

Reply:
nope, AuditUpdateRecordRequest has recordId and AuditCreateRecordRequest doesn't